airflow/providers/google/cloud/hooks/datapipeline.py

```diff
@@ -0,0 +1,88 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains a Google Data Pipelines Hook."""
+from __future__ import annotations
+
+from googleapiclient.discovery import build
+
+from airflow.providers.google.common.hooks.base_google import (
+    GoogleBaseHook,
+)
+
+DEFAULT_DATAPIPELINE_LOCATION = "us-central1"
```

Comment:
What's the reasoning behind using a default location? I think normally this is expected to be a required parameter, isn't it?

Reply:
it isn't completely necessary because it is passed a parameter, but the Dataflow hook utilizes a default location. it was recommend we do the same because of that